Fluid/Filters/MiscFilters.cs

```diff
@@ -106,7 +107,10 @@ public static ValueTask<FluidValue> Handleize(FluidValue input, FilterArguments
                 }
             }
 
-            static bool IsCapitalLetter(char c) => c >= 'A' && c <= 'Z';
+            static bool IsCapitalLetter(char c)
```

Comment:
There might be a different with the `IsInRange` trick: https://sharplab.io/#v2:C4LglgNgNAJiDUAfAAgZgATIEzoMLoG8BYAKHXOQEYA2dAIwHsGJ0BJAZ1wEMAHMYLhAAyAU2DARAJwAUAYwAWXSelkBKdAF4AfCvRh26LRvQByAIIn0XAHYx0AHmMmAWiYDcpcphr0mLDtx8AsJiEpJYcorKslDoAK7s8gySwOgAtmDWsQlJKelcAB7q2ujScZnAqnLoALSmFuqOpeXWldIulnXmJqoeJAC+QA=

Reply:
```
| Method | Job               | Runtime  | Mean     | Error     | StdDev   | Allocated |
|------- |------------------ |--------- |---------:|----------:|---------:|----------:|
| M1     | ShortRun-.NET 8.0 | .NET 8.0 | 15.90 ns |  4.675 ns | 0.256 ns |         - |
| M2     | ShortRun-.NET 8.0 | .NET 8.0 | 22.86 ns | 44.572 ns | 2.443 ns |         - |
| M1     | ShortRun-.NET 9.0 | .NET 9.0 | 16.24 ns |  6.305 ns | 0.346 ns |         - |
| M2     | ShortRun-.NET 9.0 | .NET 9.0 | 16.32 ns |  5.215 ns | 0.286 ns |         - |
```

```c#
    private string _letters = "01234567890abcdefghijklmnopqrstuvwxyZABCDEFGHIJKLMNOPQRSTUVWXYZ";

    [Benchmark]
    public void M1()
    {
        foreach (var c in _letters.AsSpan())
        {
            IsCapitalLetter(c);
        }
    }

    [Benchmark]
    public void M2()
    {
        foreach (var c in _letters.AsSpan())
        {
            IsCapitalLetter2(c);
        }
    }
```

Reply:
Odd that it behaves badly on NET 8 🤷🏻‍♂️ thanks for checking out

Reply:
If I remove the loop it's different ...

Reply:
```
| Method | Job               | Runtime  | Mean      | Error     | StdDev    | Median    | Allocated |
|------- |------------------ |--------- |----------:|----------:|----------:|----------:|----------:|
| M1     | ShortRun-.NET 8.0 | .NET 8.0 | 0.0943 ns | 2.9791 ns | 0.1633 ns | 0.0000 ns |         - |
| M2     | ShortRun-.NET 8.0 | .NET 8.0 | 0.0015 ns | 0.0466 ns | 0.0026 ns | 0.0000 ns |         - |
| M1     | ShortRun-.NET 9.0 | .NET 9.0 | 0.0053 ns | 0.1502 ns | 0.0082 ns | 0.0011 ns |         - |
| M2     | ShortRun-.NET 9.0 | .NET 9.0 | 0.0000 ns | 0.0000 ns | 0.0000 ns | 0.0000 ns |         - |
```